package.json

```diff
@@ -154,7 +155,10 @@
     "ci:version:changeset": "changeset version",
     "ci:publish": "changeset publish",
     "ts:ref": "update-ts-references --discardComments",
-    "website": "pnpm build --filter ...@verdaccio/website"
+    "website": "pnpm build --filter ...@verdaccio/website",
+    "crowdin:upload": "crowdin upload sources --auto-update --config ./crowdin.yaml",
+    "crowdin:download": "crowdin download --config ./crowdin.yaml",
```

Comment:
do we need the secrets for this as well?

Reply:
Yes, these two:
`CROWDIN_VERDACCIO_PROJECT_ID`
`CROWDIN_VERDACCIO_API_KEY`

Reply:
done ✅